style.css

```diff
@@ -0,0 +1,854 @@
+body {
```

Comment:
- [ ] You have some gap to the right of your website for the desktop version for both pages `Home and About`. For the mobile version, you have a gap to the right and a small one to the left. Please kindly fix that 🙃 😉 





> |     | Mobile version                      | Desktop version                      |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |     | <img width="444" alt="Screen Shot 2023-02-09 at 19 02 41" src="https://user-images.githubusercontent.com/74501523/217869191-34d8d3d8-d6b1-4cf8-b293-ac887111f9d7.png"> | <img width="795" alt="Screen Shot 2023-02-09 at 19 01 52" src="https://user-images.githubusercontent.com/74501523/217869322-2f53713a-4798-498c-a898-9a86fbd87641.png"> |

Reply:
Fixed! ✅ 
This was because of browser's [user agent style sheet](https://stackoverflow.com/questions/20116148/override-user-agent-style-sheet-chrome), it adds an 8px margin by default.